integration-tests/tests/update_test.go

```diff
@@ -73,7 +73,8 @@ func (s *updateSuite) TestUpdateToSameReleaseAndChannel(c *check.C) {
 		common.Reboot(c)
 	} else if common.AfterReboot(c) {
 		common.RemoveRebootMark(c)
-		c.Assert(common.GetCurrentUbuntuCoreVersion(c) > common.GetSavedVersion(c),
-			check.Equals, true)
+		currentVersion := common.GetCurrentUbuntuCoreVersion(c)
```

Comment:
As a side note, you'll probably not be able to get rid of it anytime soon, but I'd encourage the usage of package/module/library names that are slightly more indicative of the content inside them. Without that, you or someone else will end up being unable to tell the line between what should go there or not, or what you can look for there or not.

Example of names not recommended for a package: common, info, data, util, helpers, functions, dirs, tools, ...

Example of better names: testutil, osinfo, fstools, ...